svg/elements/discard.json

```diff
@@ -132,7 +135,7 @@
                 "version_added": null
               },
               "webview_android": {
-                "version_added": null
+                "version_added": false
```

Comment:
This should match Android Chrome. https://www.chromestatus.com/features/4870172764536832

Reply:
@jpmedley 
I have no idea what do you mean. 
This line is about the `href` subfeature which was never implemented (in no Chrome).

Reply:
I mistook it for the bottom of the main feature.